Platform/MacMiniFamilyPkg/AcpiTables/FADT.aslc

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
just a note: the fact that FADT exists in both the device and device family directories is an idiosyncrasy from earlier in development, moving forward it's only expected to exist in the device *family* package, you don't need to change anything, I'll make the change all at once once it's merged